cmd/fs-v1.go

```diff
@@ -1246,6 +1246,11 @@ func (fs *FSObjects) HealBucket(ctx context.Context, bucket string, dryRun, remo
 	return madmin.HealResultItem{}, NotImplemented{}
 }
 
+// Walk - walks the bucket at prefix recursively until EOF.
```

Comment:
>until EOF

Doesn't make sense to me. I suppose the returned channel is closed.

```suggestion
// Walk the bucket at prefix recursively until all files have been returned or context has been canceled.
// The returned channel is closed when there are no more files.
```

Reply:
Eof means end of the walk stream (dirent stream) since a dirent is an fd all the way down. But will clarify.